Siarhei_Hancharenka/04.Infrastructure.Vagrant.LXC/04.Infrastructure.Vagrant.LXC.md

```diff
@@ -0,0 +1,5 @@
+#04.Infrastructure.Vagrant.LXC
```

Comment:
Can you attach LXC config here?

Reply:
Sorry, I did not make the version with lxc containers. Most likely this is an artifact of merging with the underlying repository.

Reply:
Oh. It's a missprint ((